src/main/java/org/dependencytrack/policy/cel/CelPolicyLibrary.java

```diff
@@ -45,6 +52,9 @@ class CelPolicyLibrary implements Library {
     static final String VAR_VULNERABILITIES = "vulns";
 
     static final Type TYPE_COMPONENT = Decls.newObjectType(Component.getDescriptor().getFullName());
+    static final Type TYPE_INTEGRITY_META_COMPONENT = Decls.newObjectType(IntegrityMetaComponent.class.getName());
+
+    static final Type TYPE_REPOSITORY_META_COMPONENT = Decls.newObjectType(RepositoryMetaComponent.class.getName());
```

Comment:
These two are not Protobuf types, and I think we don't necessarily need them anywhere.